### ISSUE TITLE
Investigate when an entry on production was modified by user or curator

### ISSUE THREAD

**Issue by brindakv** [informatics-isi-edu/protein-database#164]
RID `2-TK8P` shows an error during release although the entry was not set to release by curator. Depositor doesn't have edit access at this stage of the workflow.

**Comment by brindakv** [comment]
This is because the error status uses the wrong terminology. 

From @svoinea: The `SUBMISSION COMPLETE` step, in addition to generate the accession code, is also running now the code to generate the exported mmCIF file (previously that was done only in the `RELEASE READY` status).
In the case of an error in generating the mmCIF file, the workflow status is changed to `ERROR` and the Process_Status to `Error: releasing entry`.
In case of success, the workflow status is changed to `HOLD`.
The code of generating the mmCIF file is then re-executed (with some minor additions) in the last step `RELEASE READY`.
In other words, that entry was never in the status `HOLD`.

**Comment by brindakv** [comment]
@brindakv to test.
